app/apps/manifest.rb

```diff
@@ -53,6 +54,11 @@ def defaults
     {"name" => "", "description" => "", "category" => "", "subcategory" => "" , "icon" => "", "role" => "", "url" => ""}
   end
 
+  # Returns the contents of the Manifest object as YAML without the preceding `!ruby/object:Manifest`
```

Comment:
In regards to a private convo with @ericfranz. `ActiveModel` does come with serialization features:

http://guides.rubyonrails.org/active_model_basics.html#serialization